admin/app/models/certification-issue-report.js

```diff
@@ -71,7 +71,7 @@ export const subcategoryToLabel = {
   [certificationIssueReportSubcategories.UNINTENTIONAL_FOCUS_OUT]:
     'Le candidat a été contraint de cliquer en dehors du cadre autorisé pour une question en mode focus',
   [certificationIssueReportSubcategories.SKIP_ON_OOPS]:
-    'Une page affichant “Oups une erreur est survenue” a contraint le candidat à ne pas répondre à la question',
+    'Une page affichant « Oups une erreur est survenue», ou tout autre problème technique lié à la plateforme a contraint le candidat à ne pas répondre à la question',
```

Comment:
```suggestion
    'Une page affichant « Oups une erreur est survenue », ou tout autre problème technique lié à la plateforme a contraint le candidat à ne pas répondre à la question',
```
Il manque un petit espace.